examples/README.md

```diff
@@ -0,0 +1,38 @@
+# Examples
+## Modifying the default HiveMQ K8s Image Non-Root Image
+Security is important [[1](https://snyk.io/blog/10-kubernetes-security-context-settings-you-should-understand)] when running containers in Kubernetes. 
+For this reason we provide the documentation on how to modify the HiveMQ image without increasing its size.
```

Comment:
... how to create a HiveMQ image with non-root user and group without increasing the size of the image.